tests/integration_tests/performance/test_boottime.py

```diff
@@ -136,20 +126,9 @@ def _configure_vm(microvm, network_info=None, initrd=False):
             network_info["iface_id"]
         )
 
-    # Configure logging.
-    log_fifo_path = os.path.join(
-        microvm.path,
-        'log_fifo' + microvm.id.split('-')[0]
-    )
-    log_fifo = log_tools.Fifo(log_fifo_path)
-
-    response = microvm.logger.put(
-        log_path=microvm.create_jailed_resource(log_fifo.path),
-        level='Info',
-        show_level=False,
-        show_log_origin=False
-    )
-    assert microvm.api_session.is_status_no_content(response.status_code)
-
     microvm.start()
-    return log_fifo, _tap if network_info else None
+    if initrd:
+        time.sleep(0.8)
+    else:
+        time.sleep(0.4)
+    return microvm.log_data, _tap if network_info else None
```

Comment:
The tests can access themselves `microvm.log_data` at any time, right?

Reply:
True, I was under the impression that I have to return the `microvm` here for changes to be propagated. That's why I've moved the particular `time.sleep()`s here and returned instead `log_data`. But you are right,`log_data` can accessed also in the caller functions :thumbsup:. I've also moved back those `sleep()`s.